make-dist

```diff
@@ -147,6 +147,15 @@ build_dashboard_frontend() {
   tar cf dashboard_frontend.tar $outfile/src/pybind/mgr/dashboard/frontend/dist
 }
 
+build_mgr_pip_pkgs() {
+  CURR_DIR=`pwd`
+
+  cd src/pybind/mgr/
+  pip install -r requirements-mgr.txt --find-links deps --target deps 
```

Comment:
Is this installing from and to the same dir? Maybe we should split those (`pip_download` and `deps`)?

BTW, we should add these new dirs to the `.gitignore` file, or we'll be fed up of seeing those files as uncomitted...

Reply:
> Is this installing from and to the same dir? Maybe we should split those (pip_download and deps)?

Yup, that way the mgr folder is not cluttered with new folders and also you don't have to coordinate cmake and install-deps. `pip install -r r..` alone will do the same actually.

Reply:
> BTW, we should add these new dirs to the .gitignore file, or we'll be fed up of seeing those files as uncomitted...

true

Reply:
Have you tried building the packages as [described here](https://docs.ceph.com/en/latest/install/build-ceph/#build-ceph-packages)? And also pushing these to ceph-ci and checking whether shaman packages are built?

Also keep in mind that AFAICS `install-deps.sh` is not called in the `make-dist` script (I also checked in the ceph-build Jenkins scripts), so probably we need to actually download the packages here.

Reply:
https://shaman.ceph.com/builds/ceph/wip-pereman2-testing-2022-09-05-1259/bc6efe9c6b458ac49c32eafeeedd1e24f13976fd/default/316822/